### DISCUSSION TITLE
How do I run from master?

### DISCUSSION THREAD

**Discussion by kevgrig** [jellyfin/jellyfin#10368] [Q&A] [answered]
I'm currently running the server and web client on branch `release-10.8.z`; however, I'm hitting an issue that [seems to be fixed in `master`](https://github.com/jellyfin/jellyfin/issues/8845#issuecomment-1747635067). What's the process to run from master instead? I understand I'll be in a less stable state but I'm a developer so I'm comfortable with that. Will the following work?

1. Back up datadir, cachedir, and configdir
2. Check out and pull `master`
3. Re-build

I also run the AndroidTV client. I've built it before and pushed it to my TV so should I build that from `master` as well?

**Comment by nielsvanvelzen** [comment, marked as answer]
Yes that's pretty much it. The master branch is what will eventually be 10.9.0. Good to note is that you **cannot downgrade back to 10.8.z** after switching to the master branch.

For the Android TV app: the master branch targets 10.8.1 and newer (With the API specification from 10.8.11) and the master branch is nearly identical to the latest beta release. So no need to build that yourself.
